### PR TITLE
Azure vm public facing

### PR DESCRIPTION
**Pull request by GauravChawda**

### CODE REVIEW

File: cartography/data/jobs/analysis/azure_vm_asset_exposure.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+			"iterative": false
+		},
+		{
+			"query": "MATCH (:AzureNetworkRoute{address_prefix: '0.0.0.0/0', next_hop_type:'Internet'})<-[:CONTAIN]-(:AzureRouteTable)<-[:CONTAIN]-(group:AzureNetworkSubnet)-[:MEMBER_NETWORK_SECURITY_GROUP]->(:AzureNetworkSecurityGroup)<-[:MEMBER_NETWORK_SECURITY_GROUP]-(:AzureNetworkInterface)<-[:MEMBER_NETWORK_INTERFACE]-(vm:AzureVirtualMachine)<-[:RESOURCE]-(:AzureSubscription{id: $AZURE_SUBSCRIPTION_ID})<-[:RESOURCE]-(:AzureTenant{id: $AZURE_TENANT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) SET vm.exposed_internet = true, (CASE WHEN NOT 'public_subnet' IN  coalesce(vm.exposed_internet_type , []) THEN vm END).exposed_internet_type = coalesce(vm.exposed_internet_type , []) + 'public_subnet';",
+			"iterative": false
+		}
+	],
+	"name": "Azure virtual machines asset internet exposure"
```
Comment (ganeshjagdale9545) [inline cartography/data/jobs/analysis/azure_vm_asset_exposure.json:57]: Implement for security groups also

**Review by ganeshjagdale9545 — CHANGES_REQUESTED** [review]
Implement for security groups also

**Review by ganeshjagdale9545 — APPROVED** [review]
Looks good

**Review by mpurusottamc — APPROVED** [review]
looks good.

**Pull request merged**